opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/client/decorator/RestTemplateMDCSpanDecorator.java

```diff
@@ -0,0 +1,39 @@
+package io.opentracing.contrib.spring.web.client.decorator;
+
+import io.opentracing.Span;
+import io.opentracing.contrib.mdc.MDCSpanDecorator;
```

Comment:
I think you forgot to add a dependency on this jar.

Reply:
It would be added once the java-web-servlet PR is merged: https://github.com/opentracing-contrib/java-web-servlet-filter/blob/0ad6fddb87bbfb820edf1caaa9242832f7088377/opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/mdc/MDCSpanDecorator.java